packages/nextjs/app/builders/0x2d90C8bE0Df1BA58a66282bc4Ed03b330eBD7911/page.tsx

```diff
@@ -0,0 +1,191 @@
+                </p>
+              </div>
+
+              {/* Skills chips */}
+              <div className="mt-5">
+                <div className="flex items-center gap-2 mb-3">
+                  {}
+                  {/* @ts-ignore */}
+                  <FaCode className="text-purple-400 w-4 h-4" />
+                  <h2 className="text-base font-semibold text-white">Skills</h2>
+                </div>
+                <div className="flex flex-wrap gap-2 justify-center">
+                  {skills.map(skill => (
+                    <span
+                      key={skill}
+                      className="px-3 py-1 text-sm rounded-full bg-gray-700 text-gray-200 border border-gray-600"
+                    >
+                      {skill}
+                    </span>
+                  ))}
+                </div>
+              </div>
+
+              {/* Social links */}
+              <div className="mt-6 flex justify-center gap-5">
+                <a
+                  href="https://github.com/Josetic224"
+                  target="_blank"
+                  rel="noopener noreferrer"
+                  className="social-link"
+                  aria-label="GitHub Profile"
+                >
+                  {}
+                  {/* @ts-ignore */}
+                  <FaGithub className="w-6 h-6" />
+                </a>
+                <a
+                  href="https://x.com/yoseph_Eth"
+                  target="_blank"
+                  rel="noopener noreferrer"
+                  className="social-link"
+                  aria-label="X Profile"
+                >
+                  {}
+                  {/* @ts-ignore */}
+                  <SiX className="w-5 h-5" />
+                </a>
+                <a
+                  href="https://www.linkedin.com/in/joseph-ochiagha-749853317"
+                  target="_blank"
+                  rel="noopener noreferrer"
+                  className="social-link"
+                  aria-label="LinkedIn Profile"
+                >
+                  {}
+                  {/* @ts-ignore */}
+                  <FaLinkedin className="w-6 h-6" />
+                </a>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+
+      {/* Styling */}
+      <style jsx global>{`
+        /* Remove scrollbar from the entire document */
+        html,
+        body {
+          overflow: hidden;
+          height: 100%;
+          margin: 0;
+          padding: 0;
+        }
+
+        .social-link {
+          display: flex;
+          align-items: center;
+          justify-content: center;
+          width: 2.5rem;
+          height: 2.5rem;
+          border-radius: 9999px;
+          background-color: #2d3748;
+          color: #e2e8f0;
+          transition: none; /* Remove transition to prevent movement */
+        }
+
+        .social-link:hover {
+          background-color: #3182ce;
+          color: white;
+        }
+      `}</style>
```

Comment:
This project follows the utility-first CSS approach, meaning we find it easier to read and style the code by adding pre-built CSS classes directly to the HTML.
We're using Tailwind 4.1.3. I suggest checking out its documentation and the rest of the project code. I assure you the learning curve is easy.